clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -418,6 +415,7 @@ public void handle(SyncGroupResponse syncResponse,
                            RequestFuture<ByteBuffer> future) {
             Errors error = Errors.forCode(syncResponse.errorCode());
             if (error == Errors.NONE) {
+                log.info("Successfully joined group {} with generation {}", groupId, generation);
                 log.debug("Received successful sync group response for group {}: {}", groupId, syncResponse.toStruct());
```

Comment:
I guess we also need this debug log because of the `syncResponse`? Does the message still make sense given the new log statement above it?